app/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ plugins {
     alias(libs.plugins.anvil)
     alias(libs.plugins.ksp)
     alias(libs.plugins.kapt)
-    id("com.google.firebase.appdistribution") version "4.0.1"
+    alias(libs.plugins.firebaseAppDistribution)
     id("org.jetbrains.kotlinx.knit") version "0.4.0"
     id("kotlin-parcelize")
```

Comment:
You may add 
`kotlin-parcelize = { id = "org.jetbrains.kotlin.plugin.parcelize", version.ref = "kotlin" }`
to the toml.
Then use:
`alias(libs.plugins.kotlin.parcelize)`

Reply:
It does not work:
```
Error resolving plugin [id: 'org.jetbrains.kotlin.plugin.parcelize', version: '1.9.20']
> The request for this plugin could not be satisfied because the plugin is already on the classpath with an unknown version, so compatibility cannot be checked.

```

I guess this plugin is already declared.

Reply:
You're right, `alias` supplies the version spec too, we must supply only the id here:

`id(libs.plugins.kotlin.parcelize.get().pluginId)`

Reply:
OK, it works, but I think it's better to keep `id("kotlin-parcelize")` since this is more known by developers and more common to have this in the wild.

Reply:
I'm not sure why the double standard: If all the available plugins were in the toml file, then developers could just use IDE completion to browse what's available and add it. It's much more handy.
Plus with recent versions of AS you can also do the opposite: go to the toml file and do "find usages" and see wherever a plugin/library is used.
But I understand this may take some time to "sink in" so it's completely not a problem if you want to save it for a later moment.